docs/template-vm.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
```

Comment:
Here the annotation is missing to tell MIQ that this is a tempalte @yanirq can you provide the right annotation?

Reply:
@fabiand @fromanirh 
see https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/8
I used a label for that matter: 

labels:
  //vm-template label applied to all objects created from this template
  template: “miq.github.io/kubvirt/vm-template”

Reply:
sure, the annotation is above, will move here

Reply:
according to the discussion in https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/8, I added the ^metadata.label$ above. Also, according to https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/8 looks like we need to use a different format, with only one "slash".